pkg/provider/resource_beacon_analytics.go

```diff
@@ -0,0 +1,306 @@
+				Attributes: map[string]schema.Attribute{
+					"nodes": schema.Int64Attribute{
+						MarkdownDescription: "Node count.",
+						Required:            true,
+					},
+					"id": schema.StringAttribute{
+						MarkdownDescription: "Cluster architecture ID. For example, \"single\" or \"ha\".For Extreme High Availability clusters, please use the [biganimal_pgd](https://registry.terraform.io/providers/EnterpriseDB/biganimal/latest/docs/resources/pgd) resource.",
+						Required:            true,
+						Validators: []validator.String{
+							stringvalidator.OneOf("single", "ha"),
+						},
+					},
+					"name": schema.StringAttribute{
+						MarkdownDescription: "Name.",
+						Optional:            true,
+						Computed:            true,
+						PlanModifiers:       []planmodifier.String{stringplanmodifier.UseStateForUnknown()},
+					},
+				},
+			},
+		},
+		Attributes: map[string]schema.Attribute{
+			"id": schema.StringAttribute{
+				MarkdownDescription: "Resource ID of the cluster.",
+				Computed:            true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.UseStateForUnknown(),
+				},
+			},
+			"cluster_id": schema.StringAttribute{
+				MarkdownDescription: "Cluster ID.",
+				Computed:            true,
+				PlanModifiers:       []planmodifier.String{stringplanmodifier.UseStateForUnknown()},
+			},
+			"connection_uri": schema.StringAttribute{
+				MarkdownDescription: "Cluster connection URI.",
+				Computed:            true,
+				PlanModifiers:       []planmodifier.String{plan_modifier.CustomConnection()},
+			},
+			"cluster_name": schema.StringAttribute{
+				MarkdownDescription: "Name of the cluster.",
+				Required:            true,
+			},
+			"phase": schema.StringAttribute{
+				MarkdownDescription: "Current phase of the cluster.",
+				Computed:            true,
+				PlanModifiers: []planmodifier.String{
+					plan_modifier.CustomPhaseForUnknown(),
+				},
+			},
+			"project_id": schema.StringAttribute{
+				MarkdownDescription: "BigAnimal Project ID.",
+				Required:            true,
+				Validators: []validator.String{
+					ProjectIdValidator(),
+				},
+			},
+			"logs_url": schema.StringAttribute{
+				MarkdownDescription: "The URL to find the logs of this cluster.",
+				Computed:            true,
+				PlanModifiers:       []planmodifier.String{stringplanmodifier.UseStateForUnknown()},
+			},
+			"backup_retention_period": schema.StringAttribute{
+				MarkdownDescription: "Backup retention period. For example, \"7d\", \"2w\", or \"3m\".",
+				Optional:            true,
+				Computed:            true,
+				Validators: []validator.String{
+					BackupRetentionPeriodValidator(),
+				},
+				PlanModifiers: []planmodifier.String{stringplanmodifier.UseStateForUnknown()},
+			},
+			"cloud_provider": schema.StringAttribute{
+				Description: "Cloud provider. For example, \"aws\", \"azure\", \"gcp\" or \"bah:aws\", \"bah:gcp\".",
```

Comment:
need to check which providers are we supported as of yet for AC and update the description

Reply:
Looks like aws only atm https://github.com/EnterpriseDB/upm-rfcs/pull/461/files 

> AWS will be the initial provider that is supported with the analytics cluster, with follow on work for supporting Azure and GCP. 

I think we can just leave aws as the example